terraform/main.tf

```diff
@@ -21,15 +21,13 @@ resource "digitalocean_droplet" "bounty_snake_droplet" {
   region      = "sfo2"
   size        = "s-2vcpu-4gb"
   monitoring  = true
-  ssh_keys    = [
-    25059594 # brandonb@echosec.net
-  ]
+  ssh_keys    = var.ssh_key_ids
   user_data   = <<EOM
     #cloud-config
     runcmd:
       - docker login docker.pkg.github.com -u ${var.github_username} -p ${var.github_token}
       - docker pull docker.pkg.github.com/echosec/bounty-snake-2020/bounty-snake-2020:${var.image_tag}
-      - docker run -t -d -p 80:5000 --env VERSION=${var.image_tag} --restart=unless-stopped docker.pkg.github.com/echosec/bounty-snake-2020/bounty-snake-2020:${var.image_tag}
+      - docker run -t -d -p 80:5000 --env VERSION=${var.image_tag} --env REDIS_HOST=${digitalocean_droplet.bounty_snake_redis.ipv4_address_private} --restart=unless-stopped docker.pkg.github.com/echosec/bounty-snake-2020/bounty-snake-2020:${var.image_tag}
```

Comment:
I've recommended a change to #37 to include the config for the `redis.createClient()` init.

```js
const REDIS_HOST = process.env.REDIS_HOST || 'localhost';
const REDIS_PORT = process.env.REDIS_PORT || 6379;
const client = redis.createClient(REDIS_PORT, REDIS_HOST);
```